.github/workflows/ci.yml

```diff
@@ -0,0 +1,139 @@
+name: CI
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
```

Comment:
Why rerun the workflow on labelling? (I didn't see specific label checks in this changeset)